docs/tutorials/oss-to-enterprise-migration-new.md

```diff
@@ -0,0 +1,99 @@
+* Start your new Rundeck enterprise instance by running the following command:
+```      
+        service rundeckd restart
+```
+* Once in Rundeck, import the project archive files that you created in step 2. Create a new project, go to project settings and select "Import Archive." On the next page, you will be able to choose what all you want imported (Jobs, Executions, Configurations, etc.)
+* Install license by choosing the license key file that was sent to you by a Rundeck team member. 
+* Set execution mode to active
+* Go to cluster manager, select  jobs, and select all orphan jobs and click action button/Change Job Schedule Owner and assign to a cluster member.
+
+## Debian/Ubuntu Upgrade
+
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
+* Export project setting from Rundeck Open Source. 
+* Stop your Rundeck Open Source using the following command:
+```
+        service rundeckd stop
+```
+* Delete Rundeck Open Source using the following command:
+```
+        apt remove rundeck
+```
+* Install Rundeck Enterprise:
+```
+        apt install rundeckpro-enterprise
+```
+* Do not edit the profile file, add your custom config from it (if any) to /etc/default/rundeckd (create the file if it is not there)
+* Start Rundeck Enterprise:
+```
+        service rundeckd restart
+```
+* Once in Rundeck, import the project archive files that you created in step 2. Create a new project, go to project settings and select "Import Archive." On the next page, you will be able to choose what all you want imported (Jobs, Executions, Configurations, etc.)
+* Install license. Navigate to Rundeck and in the top right corner click the “No License Key.” On the next page, upload the file that contains your license key. If you do not have one, reach out to a Rundeck team member with your UUID.
+* Set execution mode to active. Click on the gears icon in the top right corner and select “Execution Mode.” On the next page, ensure that execution mode is set to Active.
+* Go to cluster manager, select the jobs tab, select all orphan jobs that you wish to assign to a cluster and click action button/Change Job Schedule Owner and assign to a cluster member
+
+# WAR Upgrade
+
+* Download the executable War from [here](https://download.rundeck.com/versions.html).
+* Delete job execution history. If you do not, then the exporting of projects will take a much longer time.
+* Export project setting from Rundeck Open Source. 
+* Now, make sure that Rundeck Open Source is no longer running. 
+* After the war is downloaded, define RDECK_BASE environment variable to the location of the install:
+```
+        export RDECK_BASE=$HOME/rundeck; # or where you like it
+```
+* Create the directory for the installation:
+```
+        mkdir -p $RDECK_BASE
+```
+* Copy the executable war to the installation directory:
+```
+        cp rundeck-3.3.2-20200817.war $RDECK_BASE
```

Comment:
this line also needs {{{rundeckVersionFull}}} right?

Reply:
yes

Reply:
Updated